Samples/HelloWorld/HelloWorld/HelloWorld.csproj

```diff
@@ -12,12 +12,14 @@
     <AssemblyName>HelloWorld</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+    <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\</SolutionDir>
+    <RestorePackages>true</RestorePackages>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>$(OrleansSDK)\LocalSilo\</OutputPath>
+    <OutputPath>bin\$(Configuration)\</OutputPath>
```

Comment:
If you are just using the packages downloaded from NuGet.org, then OrleansSDK may not be defined. The DevTest Silo can run in-place from anywhere, and does not need to be conflated with LocalSilo [if the full SDK installed]  With this change, the "client" program now starts silo in background from its own bin directory.